app/src/main/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentSheet.kt

```diff
@@ -246,4 +248,14 @@ class ScheduleAppointmentSheet : BottomSheetActivity(), ScheduleAppointmentUi, S
   override fun hideProgress() {
     doneButton.setButtonState(Enabled)
   }
+
+  override fun showDoneButton() {
+    doneButton.visibility = VISIBLE
+    nextButton.visibility = GONE
+  }
+
+  override fun showNextButton() {
+    nextButton.visibility = VISIBLE
+    doneButton.visibility = GONE
```

Comment:
Same comment as above, we would want this to be a separate method called `hideDoneButton`